.circleci/config.yml

```diff
@@ -8,12 +8,23 @@ jobs:
       - checkout
       - run: yarn install --non-interactive
       - run: yarn lint
-      - run: yarn test-unit
+      - run: yarn test:unit
       - run: yarn build
       - run:
           name: publish gh pages for a given PR
           command: ./tools/deploy-gh-pages-pr.sh
 
+  test-e2e:
+    docker:
+      - image: circleci/node:12.16.1
```

Comment:
couldn't you use `cypress/included:4.1.0` image here and run commands directly without Dockerfile and without docker-compose?

Reply:
I'm using docker cause I want to have the same test result independent on the OS/machine it's running in opposite to germini/hermione tests where we have got different results on the CI than on our local machines